context-js/src/lib.rs

```diff
@@ -22,8 +25,15 @@ pub fn env_parse(env_vars: js_sys::Object) -> Option<env::parser::CIInfo> {
             }
         })
         .collect();
+
+    let stable_branches_unwrapped = stable_branches.unwrap_or_default();
+    let stable_branches_ref: &[&str] = &stable_branches_unwrapped
+        .iter()
+        .map(String::as_str)
+        .collect::<Vec<&str>>();
```

Comment:
thought it made more sense to send an empty array over an optional but let me know if we should keep using optionals instead

Reply:
Yeah, defaults to main/master if the stable branches array is empty. I can update this to use Optionals all the way (and coalesce in BranchClass::From)

Reply:
Actually, planning on making this non-optional and defaulting to main/master in the etl. i.e., the stable_branches array should never be empty